prac_07/project_management.py

```diff
@@ -0,0 +1,31 @@
+"""
+
```

Comment:
docstrings

Reply:
For all parts as well. I know it gets reapetive but have to consider its future proofness. Coders in the workplace spend 80% of their time trying to understand code and the rest actually writting something. (At least thats what I was told in my embedded systems class)